src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -78,7 +78,7 @@ public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQ
 
         try {
             return $this->handleRaw($request, $type);
-        } catch (\Exception $e) {
+        } catch (\Throwable $e) {
```

Comment:
Maybe: 

```php
catch(\Throwable | \Exception) {
```

?

Reply:
Not actually @Guikingone, back in time PHP allowed us to throw `\Exception` only, PHP7 introduced the ability to also throw `\Error`. Both now implement `\Throwable`, which means any `\Exception` will be catch by `\Throwable`.

In other words, `\Throwable` is the same of `\Exception | \Error`.